docs/Demo/index.js

```diff
@@ -52,16 +52,18 @@ const Demo = props => {
               DEMO_renderAfter,
               DEMO_renderBefore,
               DEMO_hide_renderAfterBefore = false,
+              DEMO_key,
               ...thinState
             } = state;
             const componentMarkup = renderState(thinState);
+            const key = DEMO_key || componentMarkup;
             const afterMarkup =
               DEMO_renderAfter &&
               !DEMO_hide_renderAfterBefore &&
               jsxStringify(DEMO_renderAfter, stringifyConfig);
 
             return (
-              <div key={componentMarkup}>
+              <div key={key}>
```

Comment:
Focus wasn't being maintained at all when paging, whether you were on the last page or not. That's because the DOM was getting remounted every time the markup changed. I added the ability to provide a key